sdk/eventhub/event-hubs/samples/v5/javascript/package.json

```diff
@@ -28,7 +28,7 @@
     "@azure/event-hubs": "latest",
     "dotenv": "latest",
     "rhea-promise": "^2.0.0",
-    "@azure/identity": "2.0.0-beta.5",
+    "@azure/identity": "2.0.0-beta.6",
```

Comment:
Samples should not take a dependency on a version that is not published yet. Users should be able to download the sample and run `npm install` without any issues.

Reply:
thank you! I’m not sure how we got here then 🤔 I’ll fix it! much appreciated